assets/quick-add-bulk.js

```diff
@@ -80,10 +89,11 @@ class QuickAddBulk extends HTMLElement {
         const parsedState = JSON.parse(state);
 
         if (parsedState.description || parsedState.errors) {
-          // const errorElement = document.querySelector(`#Quick-add-bulk-item-error-desktop-${event.target.getAttribute('data-index')}`)
-          // errorElement.classList.remove('hidden')
-          // errorElement.querySelector('.variant-bulk__error-text').innerHTML = parsedState.description
-         // Update errors
+          const errorElement = document.querySelector(`#quick-add-bulk-${event.target.getAttribute('data-index')} .quick-add-bulk-no-variants__error`);
+          errorElement.classList.remove('hidden');
+          this.resetQuantityInput(event.target.id);
+          this.querySelector('.progress-bar-container').classList.add('hidden');
```

Comment:
Can we save this in a variable?

`this.progressBar = this.querySelector('.progress-bar-container')`